packages/framework/src/Model/Product/Recalculation/ProductRecalculationDispatcher.php

```diff
@@ -5,25 +5,9 @@
 namespace Shopsys\FrameworkBundle\Model\Product\Recalculation;
 
 use Shopsys\FrameworkBundle\Component\Messenger\AbstractMessageDispatcher;
-use Shopsys\FrameworkBundle\Model\Product\Product;
 
 class ProductRecalculationDispatcher extends AbstractMessageDispatcher
 {
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Product\Product[] $products
-     * @param \Shopsys\FrameworkBundle\Model\Product\Recalculation\ProductRecalculationPriorityEnum $productRecalculationPriorityEnum
-     * @return int[]
-     */
-    public function dispatchProducts(
```

Comment:
Even though this method is currently unused shouldn't be kept for easy use from the project?